dev-libs/bglibs/bglibs-1.041.ebuild

```diff
@@ -1,51 +0,0 @@
-# Copyright 1999-2018 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=0
-
-inherit fixheadtails toolchain-funcs multilib
-
-DESCRIPTION="Bruce Guenters Libraries Collection"
-HOMEPAGE="http://untroubled.org/bglibs/"
-SRC_URI="http://untroubled.org/bglibs/archive/${P}.tar.gz"
-
-LICENSE="LGPL-2.1"
-SLOT="0"
-KEYWORDS="alpha amd64 ~arm hppa ~mips ppc sparc x86 ~ppc64"
```

Comment:
Cannot drop this as it would remove the stable keyword for `sparc` at this point.

Reply:
I see.